Lib/posixpath.py

```diff
@@ -398,6 +398,18 @@ def abspath(path):
     return normpath(path)
 
 
+try:
+    from posix import _path_abspath
+except ImportError:
+    abspath = _abspath_fallback
+else:
+    def abspath(path):
+        """Return an absolute path."""
+        try:
+            return _path_abspath(path)
+        except (UnicodeEncodeError, ValueError):
```

Comment:
Unicode errors are subclasses of `ValueError`.

```suggestion
        except ValueError:
```